recipes/icon/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/env bash
+make Configure name=linux
+# there should be a better way than Makedefs hacking with sed...
+sed -i -e 's/CC = gcc/CC = x86_64-conda-linux-gnu-gcc/' Makedefs
+echo Build Icon, run a sample of the test suite, and install upon success
+make
```

Comment:
Probably this works better for setting the compiler 
```suggestion
make CC=${CC}
```